libdrgn/object.c

```diff
@@ -837,7 +837,7 @@ LIBDRGN_PUBLIC struct drgn_error *
 drgn_format_object(const struct drgn_object *obj, size_t columns,
 		   enum drgn_format_object_flags flags, char **ret)
 {
-	const struct drgn_language *lang = &drgn_language_c;
+	const struct drgn_language *lang = drgn_type_lang(obj->type);
```

Comment:
This is repeated often enough that I think it merits an inline `drgn_object_lang()` helper in `drgn.h`.